test/PerpetualPositionRouter.t.sol

```diff
@@ -221,22 +221,35 @@ contract Fallback is PositionRouterTest {
   }
 
   function testFork_FailedCallWhenExtraCalldataArgument() public {
+    delegateApproval.approve(vethPositionRouterAddr, 1);
```

Comment:
Why was this line added to some tests, seems they passed properly both with and without it?

Reply:
These checks are checking for failures and if the delegate has not approved the router than the call will fail. I added it to make sure the other conditions are causing a failure. Is there a better way when calling fallback to get the specific error?

Reply:
Ahh you're saying that because we're not using `expectRevert` since it's a low level call, it's possible these were bad tests, in that they were failing due to a lack of approval.

We can do this in a follow up PR, but we should also be checking the return data to make sure the revert reason matches what we expect